README.md

```diff
@@ -261,7 +283,7 @@ This service is responsible for running dynamic simulations. It is based on [Pow
 - Other services dependencies: network store server
 - Use PowSyBl libraries: yes
 
-This service is responsible for configuring how dynamic data is mapped to a network. This is still an ongoing work and is not yet integrated with dynamic simulation server.
+This service is responsible for configuring how dynamic data is mapped to a network. This is still an ongoing work and is not yet completely integrated with dynamic simulation server.
```

Comment:
same

Reply:
DONE in https://github.com/gridsuite/documentation/pull/16/commits/c536a6b68417567e27b8c6840abbf5c4a42c09dd